lms/djangoapps/teams/static/teams/templates/team-member.underscore

```diff
@@ -0,0 +1,8 @@
+
+<span class="team-member">
+    <a class="member-profile" href="<%= memberProfileUrl %>">
+        <p><%- gettext(username) %></p>
+        <img class="image-url" src="<%= imageUrl %>" alt="<%- gettext(username) %>" />
```

Comment:
`profile-image` will be a better class name. And again `username` should not be in `gettext`.